frb_codegen/src/ir/import.rs

```diff
@@ -0,0 +1,5 @@
+#[derive(Debug, Clone, Hash, PartialEq, Eq)]
+pub struct IrImport {
```

Comment:
After reading example, I realized it is not "import" in Rust, but instead in Dart. Shall we rename `IrImport -> IrDartImport` and `IrAnnotation -> IrDartAnnotation`?

Reply:
Sure. :-)